articles/quickstart/webapp/java-spring-security-mvc/_includes/_login.md

```diff
@@ -116,20 +116,21 @@ public AuthenticationController authenticationController() throws UnsupportedEnc
 }
 ```
 
-To authenticate the users you will redirect them to the login page which uses [Universal Login](https://auth0.com/docs/universal-login). This page is accessible from what we call the "Authorize URL". By using this library you can generate it with a simple method call. It will require a `HttpServletRequest` to store the call context in the session and the URI to redirect the authentication result. This URI is normally the address where your app is running plus the path where the result will be parsed, which happens to also be the "Callback URL" whitelisted before. Finally, request the scopes `openid profile email` to get back user profile information in the ID token upon login. After you create the Authorize URL, you redirect the request there so the user can enter their credentials. The following code snippet is located on the `LoginController` class of our sample:
+To enable users to login, your application will redirect them to the [Universal Login](https://auth0.com/docs/universal-login) page. Using the `AuthenticationController` instance, you can generate the redirect URL by calling the `buildAuthorizeUrl(HttpServletRequest request, HttpServletResponse response, String redirectUrl)` method. The redirect URL must be the URL that was added to the **Allowed Callback URLs** of your Auth0 Application.
```

Comment:
Reduced and simplified this paragraph to be consistent with other Java webapp Quickstarts explanation of generating the login URL.